public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -290,7 +290,7 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
                   ) : detector.enabled &&
                     detector.curState === DETECTOR_STATE.INIT ? (
                     <EuiHealth color={DETECTOR_STATE_COLOR.INIT}>
-                      {detector.initProgress
+                      {detector.initProgress?.estimatedMinutesLeft
```

Comment:
`estimatedMinutesLeft` means how many minutes user needs to wait for initialization to complete? For example, if user need to wait for another 5 minutes to pass initialization, `estimatedMinutesLeft` will be 5?

Reply:
`estimatedMinutesLeft` is calculated in the backend, which is num shingles needed * detector interval. Returns undefined until cold start is passed.

More details [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection/pull/210)

Reply:
Never mind, find answer in `AnomalyResults.tsx` file. We will split cases before/after cold start and show different messages.